filesystem/iso9660.ksy

```diff
@@ -216,60 +94,867 @@ types:
+          - id: volume_set_size
+            doc-ref: ecma-119 8.4.10
+            type: u2bi
+          - id: volume_sequence_number
+            doc-ref: ecma-119 8.4.11
+            type: u2bi
+          - id: logical_block_size
+            doc-ref: ecma-119 8.4.12
+            type: u2bi
+          - id: path_table_size
+            doc-ref: ecma-119 8.4.13
+            type: u4bi
+          - id: loc_l_path_table
+            doc-ref: ecma-119 8.4.14
+            type: u4le
+          - id: loc_opt_l_path_table
+            doc-ref: ecma-119 8.4.15
+            type: u4le
+          - id: loc_m_path_table
+            doc-ref: ecma-119 8.4.16
+            type: u4be
+          - id: loc_opt_m_path_table
+            doc-ref: ecma-119 8.4.17
+            type: u4be
+          - id: directory_record_for_root_directory
+            doc-ref: ecma-119 8.4.18
+            doc: |
+              The root directory_record with body is 34 bytes.
+              In other cases the size is variable.
+            type: directory_record
+            size: 0x22
+          - id: volume_set_identifier
+            doc-ref: ecma-119 8.4.19
+            type: text128
+          - id: publisher_identifier
+            doc-ref: ecma-119 8.4.20
+            type: text128
+          - id: data_preparer_identifier
+            doc-ref: ecma-119 8.4.21
+            type: text128
+          - id: application_identifier
+            doc-ref: ecma-119 8.4.22
+            type: text128
+          - id: copyright_file_identifier
+            doc-ref: ecma-119 8.4.23
+            type: text37
+          - id: abstract_file_identifier
+            doc-ref: ecma-119 8.4.24
+            type: text37
+          - id: bibliographic_file_identifier
+            doc-ref: ecma-119 8.4.25
+            type: text37
+          - id: volume_creation_date_and_time
+            doc-ref: ecma-119 8.4.26
+            type: datetime_long
+          - id: volume_modification_date_and_time
+            doc-ref: ecma-119 8.4.27
+            type: datetime_long
+          - id: volume_expiration_date_and_time
+            doc-ref: ecma-119 8.4.28
+            type: datetime_long
+          - id: volume_effective_date_and_time
+            doc-ref: ecma-119 8.4.29
+            type: datetime_long
+          - id: file_structure_version
+            doc-ref: ecma-119 8.4.31
+            type: s1
+        instances:
+          path_table:
+            io: _root._io
+            pos: _root.sector_size * loc_l_path_table
+            size: path_table_size.le
+            type: path_table_records
+      supplementary:
+        doc-ref: ecma-119 8.5
+        seq:
+          - id: volume_flags_reserved
+            doc-ref: ecma-119 8.5.3 b1-b7
+            type: b7
+          - id: volume_flags_not_iso2375
+            doc-ref: ecma-119 8.5.3 b0
+            type: b1
+          - id: system_identifier
```

Comment:
duplication

Reply:
I don't see it. One is system other is volume, even the doc-ref is different

Reply:
https://github.com/kaitai-io/kaitai_struct_formats/pull/82/files#diff-1e076bd9a91d2da636ba57110890a3857b5ebade3b60b49ed57f9d1d35dc08d2R162

Reply:
@SnowFlakey

Reply:
I have implemented these standards exactly!
The fact that ref. 8.4 & 8.5 partially contain the same items and not a reference to example a u4bi is not for me to fix.

```
      primary:
        doc-ref: ecma-119 8.4
```
```
          - id: system_identifier
            doc-ref: ecma-119 8.4.5
            type: text32
          - id: volume_identifier
            doc-ref: ecma-119 8.4.6
            type: text32
```

```
      supplementary:
        doc-ref: ecma-119 8.5
```
```
          - id: system_identifier
            doc-ref: ecma-119 8.5.4
            type: text32
          - id: volume_identifier
            doc-ref: ecma-119 8.5.5
            type: text32
```

Reply:
IMHO the fact that it is written so in the original spec shouldn't prevent us from eliminating redundancy. So, it is for you to fix, their spec is their, but the ksy spec is yours!

Reply:
Well I guess we conflict on 2 items... this and the naming of some identifiers as that topic came up 2 times.

This PR works and will greatly extent the support of the format.
If you don't want my code, then let me know and I will close the pr.

Reply:
Don't close your PR, your PR is welcome here! The notices made are not requirements, but advices.